package.json

```diff
@@ -149,47 +148,8 @@
     "not op_mini all"
   ],
   "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.{js,jsx,ts,tsx}",
-      "!src/{App,Routes,index,registerServiceWorker}.js",
-      "!src/**/*.Component.js",
-      "!src/_playground/**"
-    ],
-    "resolver": "jest-pnp-resolver",
     "setupFiles": [
-      "react-app-polyfill/jsdom",
       "./config/jest/jest.config.js"
-    ],
-    "testMatch": [
-      "<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}",
-      "<rootDir>/src/**/?(*.)(spec|test).{js,jsx,ts,tsx}"
-    ],
-    "testEnvironment": "jsdom",
-    "testURL": "http://localhost",
```

Comment:
I know we've had problems in the past when a `testURL` is not defined in nui-widgets, nui-widgets-lab and I recently had the same problem in pkg-upgrader. Should we keep this part just in case?

Reply:
i was going to wait until we had issues, and then if we did, relocate these values to `/config/test/jest/jest.config.js`

Reply:
I'm in favor of moving the config to `/config/test/jest/jest.config.js`, but I tend to agree with @jbadan that we should just move these now so someone without prior exposure to that sort of failure doesn't spend a bunch of time researching the issue.